.github/workflows/build.yml

```diff
@@ -138,6 +162,14 @@ jobs:
         run: |
           ./bin/minetest --run-unittests
 
+  docker:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
why is this here?

Reply:
i added a docker build because:
* we never test it on github, just asynchronously on gitlab
* in docker world we always have the need for prometheus metrics, and it's embedded in the docker file

Reply:
This change should be in a separate commit at least.

Reply:
ok, will do